{{cookiecutter.github_repository}}/circle.yml

```diff
@@ -0,0 +1,87 @@
+## Customize the test machine
+machine:
+  python:
+    version: {{ '3.5' if cookiecutter.python3.lower() == 'y' else '2.7.4'}} # 2.7 will cause an error with py.test
```

Comment:
I will change it to 2.7.11, but do we want to pin it to 3.5.1 when we don't pin it in the .travis.yml file?
Another option is to pin both files, in which case I would change the travis.yml to 2.7.11 as well. 
Thoughts?

To clarify: The only reason I was pinning the 2.7 version is because I had some issues when I didn't.